apple/MarkdownTextInputDecoratorComponentView.mm

```diff
@@ -115,11 +131,14 @@ - (void)didAddSubview:(UIView *)subview
   }
 }
 
-- (void)willMoveToWindow:(UIWindow *)newWindow
+- (void)removeTextInputObservers
 {
-  if (newWindow != nil) {
+  if (!_observersAdded) {
     return;
   }
```

Comment:
Why can't we have `react_native_assert(_observersAdded);` instead?

Reply:
> though I needed to revert the asserts as the attach/detach methods can be called twice now: when the view is mounted for the very first time and when it's destroyed

Reply:
Actually, I'm pretty sure we don't need to listen for window changes at all (we can keep observers attached on non-foreground screens), and listening for subview changes is enough: [3825919](https://github.com/Expensify/react-native-live-markdown/pull/689/commits/38259195031bba2c86d89b70e2e5547e0e30433c). In that case we can restore assertions.